py3status/modules/volume_status.py

```diff
@@ -94,12 +96,16 @@
 
 
 class AudioBackend():
-    def __init__(self, parent):
+    def __init__(self, parent, output_device=True):
```

Comment:
So here we are passing in `output_device` but we know this as it is `not parent.is_input`  it might be worth making things `self.is_input` to keep things the same everywhere.